libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -78,4 +79,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://www.firetext.co.uk/docs',
     logoFileName: { light: 'firetext.svg', dark: 'firetext.svg' },
   },
+  {
+    id: SmsProviderIdEnum.Bulksms,
```

Comment:
```suggestion
    id: SmsProviderIdEnum.BulkSMS,
```
We haven't an agreed naming convention regarding uppercasing but by the provider name (BulkSMS) it feels better to write it this way even as enum key. My reasoning is that for other providers like Sms77, the company brands it like `sms77` so it felt right the chosen casing.

Reply:
Yes we can do that

Reply:
This hasn't been addressed yet.

Reply:
@p-fernandez have done it and pushed it please check and inform me if any help is needed

Reply:
The suggestion hasn't been applied. Please change to:
```suggestion
    id: SmsProviderIdEnum.BulkSms,
```